pipelines/analysis_matched.yaml

```diff
@@ -8,6 +8,7 @@ tasks:
       python: |
         from metric_pipeline_tasks import PA1Task
         config.measure.retarget(PA1Task)
+        config.measure.randomSeed = 12345
```

Comment:
I know we do it in several places, but I'm a fan of not setting a config in the pipeline if it is just setting it to the default.  It clutters the config and makes people wonder if they are misunderstanding something when looking at the code

Reply:
Fair point. I will remove these three lines setting `randomSeed` in the next commit.